R/rep_len_linter.R

```diff
@@ -0,0 +1,24 @@
+#' Require usage of rep_len(x, n) over rep(x, length.out = n)
+#'
+#' `rep(x, length.out = n)` calls `rep_len(x, n)` "under the hood". The latter
+#'   is thus more direct and equally readable.
+#'
+#' @evalRd rd_tags("rep_len_linter")
+#' @seealso [linters] for a complete list of linters available in lintr.
+#' @export
+rep_len_linter <- make_linter_from_xpath(
+  # match starts-with(text(), 'len') to get common partial matching usage.
```

Comment:
This is not implemented, right?

Reply:
Oh, good catch. I wonder what happened... maybe I didn't get any new matches from using `len`? Do you think it's worth implementing?

Reply:
Not really. I remember we have an issue lying around that lints all partial argument matching. IINM that could even be default because the tidyverse style guide discourages it.